x-pack/plugins/translations/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "./target/types",
+    "emitDeclarationOnly": true,
+    "declaration": true,
+    "declarationMap": true
+  },
+  "include": ["server/**/*", "translations/ja-JP.json", "translations/zh-CN.json"],
```

Comment:
@smith why do we have the translation files included in the tsconfigs? I think it might be causing vscode's tsserver to OOM https://github.com/microsoft/vscode/issues/133515

Reply:
Ahh.. I see. We have stuff like https://github.com/elastic/kibana/blob/master/x-pack/plugins/observability/public/utils/test_helper.tsx#L17 which imports the translation json. Guess that's why it's required.

Reply:
It's probably something we can get rid of. If I remember right it's for mocking i18n in tests and can probably be done some other way. 

It would be nice if we could measure the before/after impact as well to understand the performance better.

Reply:
sure would. In this case it seems like anything that's `include`ed via tsconfig.json is fair game for VSCode tsserver analysis (rightly so I'd say). It seems like this file is probably what was causing that to OOM on the default 512mb node heap max.